Scripts/Services/CompareComply/V1/Model/FeedbackDeleted.cs

```diff
@@ -17,22 +17,22 @@
 
 using Newtonsoft.Json;
 
-namespace IBM.Watson.Discovery.V1.Model
+namespace IBM.Watson.CompareComply.V1.Model
```

Comment:
Maybe I am missing it but I am not seeing the `Discovery` Model, are we getting rid of it?

Reply:
Hmm the discovery models are there  - this might be some weird renaming thing